### PR TITLE
Fixed cameras taking pictures when you place them on a table

### PR DESCRIPTION
**Pull request by JMWTurner**
duh

### CODE REVIEW

**Comment by ComicIronic** [comment]
The real fix for this is that dropping an item on a table should cause the attackby to return 1, indicating that afterattack shouldn't be called.

**Comment by JMWTurner** [comment]
revision 1
- aye sir

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -407,7 +407,7 @@
 			var/clamp_y = clicked.Height() / 2
 			W.pixel_x = Clamp(text2num(params_list["icon-x"]) - clamp_x, -clamp_x, clamp_x)
 			W.pixel_y = Clamp(text2num(params_list["icon-y"]) - clamp_y, -clamp_y, clamp_y)
-	return
+	return 1
```
Comment (clusterfack) [inline code/game/objects/structures/tables_racks.dm]: Whoa whoa, this should be tabbed forward two spaces, no?

Comment (ComicIronic) [reply code/game/objects/structures/tables_racks.dm]: It should, the trailing return should be removed.

Comment (JMWTurner) [reply code/game/objects/structures/tables_racks.dm]: What about the one for racks at line 758 ?

Comment (clusterfack) [inline code/game/objects/structures/tables_racks.dm]: Yes that one too

**Comment by JMWTurner** [comment]
revision 2
- removed trailing returns

**Pull request merged**